### DISCUSSION TITLE
File transfer and error : ValueError: Unexpected output from check_file_exists

### DISCUSSION THREAD

**Discussion by verma-rohit80** [ktbyers/netmiko#3077] [Q&A]
I am trying to copy file from local dir to bootflash but for some odd reason it is not going though. Can anyone please have a look and suggest a work around

![file_transfer](https://user-images.githubusercontent.com/72044119/211715785-4a0b4464-799b-4853-b8ec-c7e45ae3eea7.png)

Error:
```
/home/rohit/Documents/python/Udemy/para_miko/para_virtual/PROJECT_4/venv/FILE_TRANSFER
Connected Successfully
configure terminal
Enter configuration commands, one per line.  End with CNTL/Z.
csr2(config)#
Traceback (most recent call last):
  File "/home/rohit/Documents/python/Udemy/para_miko/para_virtual/PROJECT_4/venv/FILE_TRANSFER/f_transfer.py", line 19, in <module>
    transfer = file_transfer(
  File "/home/rohit/.local/lib/python3.10/site-packages/netmiko/scp_functions.py", line 123, in file_transfer
    if scp_transfer.check_file_exists():
  File "/home/rohit/.local/lib/python3.10/site-packages/netmiko/scp_handler.py", line 236, in check_file_exists
    raise ValueError("Unexpected output from check_file_exists")
ValueError: Unexpected output from check_file_exists
```

**Comment by ktbyers** [comment]
Is this a real device or a fake GNS3/virtual device?

**Comment by verma-rohit80** [comment]
It is CSR running on eve-ng.

**Reply by ktbyers** [reply]
If you execute `dir` from the CLI what does the output look like?

**Comment by verma-rohit80** [comment]
TRANSFER$ ls -la
total 16
drwxrwxr-x 2 rohit rohit 4096 Jan 11 09:15 .
drwxrwxr-x 6 rohit rohit 4096 Jan 11 08:16 ..
-rw-rw-r-- 1 rohit rohit 1634 Jan 11 09:15 config.txt
-rw-rw-r-- 1 rohit rohit  683 Jan 11 09:28 f_transfer.py
rohit@rohit:~/Documents/python/Udemy/para_miko/para_virt

**Reply by ktbyers** [reply]
No I am looking for `dir` from the CSR CLI (i.e. when you SSH to 192.168.1.20).

Also why did you enter `config_mode` with the SSH connection (c = conn.config_mode())? You probably shouldn't do that.
